frontback/frontBackCSHORE.py

```diff
@@ -1040,832 +924,177 @@ def CSHOREsimSetup(startTime, inputDict):
-                            else:
-                                meta_dict['blank_wave_data'] = date_list[np.argwhere(dum_var).flatten()]  # this will list all the times that wave data should exist, but doesn't
-                            print "%d wave records with %d interpolated points" % (np.shape(wave_data['dWED'])[0], timerun + 1 - len(wave_data['dWED']))
-
-                            helper = np.vectorize(lambda x: x.total_seconds())
-                            BC_dict['Hs'] = np.interp(BC_dict['timebc_wave'],helper(wave_data['time'] - wave_data['time'][0]), wave_data['Hs'])  # WE ARE USING Hmo (Hs in wave_data dictionary) INSTEAD OF Hs!!!!! -> convert during infile write!!!
-                            BC_dict['Tp'] = np.interp(BC_dict['timebc_wave'],helper(wave_data['time'] - wave_data['time'][0]), np.divide(1,wave_data['peakf']))  # we are inverting the peak frequency to get peak period
-                            BC_dict['angle'] = np.interp(BC_dict['timebc_wave'],
-                                                         helper(wave_data['time'] - wave_data['time'][0]),
-                                                         geo2STWangle(wave_data['waveDp'], zeroAngle=71.8,fixanglesout=1))  # i am assuming that the angle is the peak of the directional spectra, not the MEAN ANGLE!!!  also I'm assuming 0-360 degrees!
-                            # TODO @David use a constant exposed parameter for pier angle (at the top - a global variable) this way if it needs to be changed it's changed everywhere
-                        except:
-                            # If that doesn't work, you are done....
-                            assert 'Hs' in BC_dict.keys(), 'Simulation broken.  Previous simulation ran from 6m AWAC and wave data missing 6m AWAC.!'
-
-                    else:
-                        # go through my normal decision tree
-                        # Attempt to get 8m array first!!!
-                        try:
-                            wave_data = wave_data8m
-                            meta_dict['BC_gage'] = wave_data['name']
-                            print "_________________\nGathering Wave Data from %s" % (wave_data['name'])
-
-                            # check to see if I am missing my first and last points - if so then I can't interpolate
-                            assert start_time in wave_data['time'] and end_time in wave_data['time'], 'Wave data are missing for simulation start time or end time!'
-                            # if I am missing more than 1/4 of the data I should have, abort the run
-                            assert len(wave_data['Hs']) > 0.75 * len(BC_dict['timebc_wave']), 'Missing more than 25% of wave data'
-
-                            # get missing wave data if there is any!
-                            date_list = np.array([start_time + DT.timedelta(hours=x) for x in range(0, timerun + 1)])
-                            dum_var = [x not in w
```

Comment:
issue 2, and please rename the instance to something other than dum_class.  I would prefer if we kept instances of classes standard across models standard (ie prep in this case)so it would be easier to switch between work flows, but I don't have ultra strong feelings we  can discuss

Reply:
all instances of classes PDL.PrepDataTools() have been renamed to prep.  initialization of all classes (getObs, getDataTestBed, and PrepDataTools) is now pulled outside the FIXED - MOBILE - MOBILE_RESET decision tree.